app/views/cards/_card_form.html.erb

```diff
@@ -0,0 +1,105 @@
+<%= form_for [@deck, @card] do |f| %>
+          
+  <%= render 'shared/errors', resource: @card %>
+  
+  <div class="form-group">
+    <label>Guided Quotes</label>
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" value="">
+        Esse nihil ducimus dolores voluptate veniam fuga. Consequuntur excepturi qui. Quod libero voluptatum dolore. Qui voluptate voluptates nihil ut.
+      </label>
+    </div>
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" value="">
+        Maxime porro id cum sed quae velit atque. Non quo iure sit corrupti. Sit ut cumque libero dignissimos voluptates molestias aliquam. Blanditiis maiores tenetur ipsum. Soluta perspiciatis magni est.
+      </label>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <label>Difficulty</label>
+    <select class="form-control">
+      <option>1</option>
+      <option>2</option>
+      <option selected="selected">3</option>
+    </select>
+  </div>
```

Comment:
Well probably need to put the `difficulty` and `guiding quote` parts of the form within the tabs (i think) to make it easier to associate those fields with the unique form for each card type